OracleSOASuite/dockerfiles/README.md

```diff
@@ -108,109 +41,22 @@ Verify you now have the image `oracle/soa:12.2.1.3` in place with
-
-
-## Creating a container for SOA Server
+# Building Docker Image with bundle patches for SOA
+IMPORTANT: To build the SOA image with bundle patches, you must first download the required version of the Oracle SOA Suite binaries as explained in above step.
 
-Start a container to launch the Managed Server from the image created. The environment variables used to run the Managed Server image are defined in the file `soaserver.env.list`. 
+Download and drop the patch zip files (for e.g. `p29928100_122130_Generic.zip`) into the `patches/` folder under the version which is required, for e.g. for `12.2.1.3.0` the folder is `12.2.1.3/patches`
 
-Create an environment file `soaserver.env.list` with the below details:
-
-    MANAGED_SERVER=<managed server name, For Exp:- soa_server1, osb_server1, bpm_server1>
-    DOMAIN_TYPE=<soa/osb/bpm>
-    ADMIN_HOST=<Admin host name>
-    ADMIN_PORT=<port number where Admin Server is running>
-    
-Sample Data will look like this:
-
-    MANAGED_SERVER=soa_server1
-    DOMAIN_TYPE=soa
-    ADMIN_HOST=slc09cwi.us.oracle.com
-    ADMIN_PORT=7001
-    
-To start a docker container for SOA server you can simply call `docker run` command and passing `soaserver.env.list`. 
-
-A sample docker run command is given below:
-
-    $ docker run -i -t  --name soams --network=SOANet -p 8001:8001   --volumes-from soaas   --env-file ./soaserver.env.list oracle/soa:12.2.1.3 "/u01/oracle/dockertools/startMS.sh"
-    
-Using `--volumes-from` reuses the volume created by the Admin container. In the above `docker run` command, `soaas` is the name of the Admin server container started in the previous step and we must use the same name used for the AdminServer to start the SOA managed server. 
-
-This includes the command to tail logs to keep the container up and running.
-
->IMPORTANT: You need to wait till all the above commands are run before you can start the SOA Server.
-
-The following lines highlight when the SOA managed server is ready to be used:    
-`INFO: Managed Server is running`
-
-`INFO: Managed server has been started`
-
-Once the SOA container is created logs will be tailed and displayed to keep the container running.
-
-Now you can access
-
-   * AdminServer Web Console at http://<hostname>:7001/console  with weblogic/Welcome1 credentials.
-
-   * EM Console at http://<hostname>:7001/em  with weblogic/Welcome1 credentials.
-
-   * SOA infra Console at http://<hostname>:8001/soa-infra with weblogic/Welcome1 credentials.
-
-   * Service Bus Console at http://<hostname>:7001/servicebus with weblogic/Welcome1 credentials.
+Build the image with the -p option which will copy and install the patches into the image.
 
+$ sh buildDockerImage.sh -v 12.2.1.3 -p
```

Comment:
How do you build a patched image _without_ using the `buildDockerImage.sh` script?